source/docs/software/pathplanning/index.rst

```diff
@@ -7,8 +7,11 @@ Notice on Swerve Support
 ------------------------
 
 Swerve support in path following has a couple of limitations that teams need to be aware of:
+
 - WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.
+
```

Comment:
FWIW I believe the blank lines between the dot points aren't required. The blank line to separate the paragraph from the list is required though.